app/services/places_service.rb

```diff
@@ -3,8 +3,10 @@ def self.conn
     Faraday.new(url: 'https://api.geoapify.com', params: { apiKey: ENV['places_api_key']})
   end
 
-  def self.get_hotels(city)
-    response = conn.get("v2/places?categories=accommodation.hotel&filter=circle:#{city.longitude},#{city.latitude},20000")
+  default_categories = []
+
+  def self.get_hotels(city_info, caetgory = default_categories)
```

Comment:
Typo in category, I'll fix in my PR

Reply:
Already fixed! circleCI caught the unintentional inclusion of this from our conversation.